package.json

```diff
@@ -52,7 +52,8 @@
           "rum-angular",
           "rum-core",
           "rum-react",
-          "rum-vue"
+          "rum-vue",
+          "testing"
```

Comment:
Please let me know if this new scope is acceptable

Reply:
We use `chore` or `test` for the test changes as per the conventional commit suggestion. 

```
types other than fix: and feat: are allowed, for example @commitlint/config-conventional (based on the the Angular convention) recommends build:, chore:, ci:, docs:, style:, refactor:, perf:, test:, and others.
```

Reply:
Is it acceptable if I rewrite commit history changing the three commit messages?

Reply:
We squash and merge anyway, so feel free to do it when squash merging the PR in the UI itself.

Reply:
So let me add a commit on top reverting the addition of the new scope in Husky, thanks!

Reply:
At the moment I reverted it and pushed, lint failed. I pushed force the branch with the commit messages updated